src/js/components/Alert.js

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import Icon from './Icon';
+
+const Alert = ({children, showIcon, type}) => {
+  const classes = classNames('alert', {
+    [`alert-${type}`]: type != null
+  });
+  let icon = null;
+
+  if (showIcon) {
+    const className = 'alert-icon';
+    const family = 'mini';
+    const size = 'mini';
+
+    switch (type) {
```

Comment:
Or maybe even extract `danger` and `success` as constants and operate this constants and not bare string. In this case we don't need a switch at all.
```
const DANGER = 'ring-exclamation';
const SUCCESS = 'checkmark';

....

<Alert type={Alert.SUCCESS}>
```